docs/examples/events/dictionary.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
+
+try:
+    import httpx
+except ImportError:
+    raise ImportError("Please install http with 'pip install httpx' ")
+
+from rich.json import JSON
+
+from textual.app import App, ComposeResult
+from textual.layout import Vertical
+from textual.widgets import Static, TextInput
+
+
+class DictionaryApp(App):
+    """Searches ab dictionary API as-you-type."""
+
+    def compose(self) -> ComposeResult:
+        yield TextInput(placeholder="Search for a word")
+        yield Vertical(Static(id="results", fluid=False), id="results-container")
+
+    async def on_text_input_changed(self, message: TextInput.Changed) -> None:
+        """A coroutine to handle a text changed message."""
+        if message.value:
+            # Look up the word in the background
+            asyncio.create_task(self.lookup_word(message.value))
+        else:
+            # Clear the results
+            self.query_one("#results", Static).update()
+
+    async def lookup_word(self, word: str) -> None:
+        """Looks up a word."""
+        url = f"https://api.dictionaryapi.dev/api/v2/entries/en/{word}"
+        async with httpx.AsyncClient() as client:
+            results = (await client.get(url)).text
+
+        if word == self.query_one(TextInput).value:
+            self.query_one("#results", Static).update(JSON(results))
+
+
+app = DictionaryApp(css_path="dictionary.css")
+if __name__ == "__main__":
+    app.run()
```

Comment:
Cases of what? Are you conflating this with the CSS_PATH pr?

Reply:
Was meaning that if you merge this and the CSS_PATH PR, then the `css` branch will be broken since this code isn't compatible. Unless I've missed something.

Reply:
Ah, the css_path PR was branched from this branch. I confused you by requesting reviews in reverse order.

Reply:
Ah I see, didn't catch that. Thanks :)